File: Tarea4 (y 5 juntas)_LeyvaBanda/Tarea4.cs
using System;

class Program {
    static void Main() {
        int[,] matriz = {
            {1, 2, 3},
            {4, 5, 6},
            {7, 8, 9}
        };

        Console.WriteLine("Matriz original:");
        for (int i = 0; i < 3; i++) {
            Console.Write("[ ");
            for (int j = 0; j < 3; j++) {
                Console.Write(matriz[i, j] + " ");
            }
            Console.WriteLine("]");
        }

        Console.WriteLine("\nMatriz en filas:");
        for (int i = 0; i < 3; i++) {
            for (int j = 0; j < 3; j++) {
                Console.Write(matriz[i, j] + " ");
            }
        }

        Console.WriteLine("\nMatriz en columnas:");
        for (int j = 0; j < 3; j++) {
            for (int i = 0; i < 3; i++) {
                Console.Write(matriz[i, j] + " ");
            }
        }
    }
}

File: Tarea4 (y 5 juntas)_LeyvaBanda/Tarea4.js
let matriz = [
  [1, 2, 3],
  [4, 5, 6],
  [7, 8, 9]
];

console.log("Matriz original:");
for (let i = 0; i < 3; i++) {
  let fila = "[ ";
  for (let j = 0; j < 3; j++) {
    fila += matriz[i][j] + " ";
  }
  fila += "]";
  console.log(fila);
}

console.log("\nMatriz en filas:");
for (let i = 0; i < 3; i++) {
  for (let j = 0; j < 3; j++) {
    process.stdout.write(matriz[i][j] + " ");
  }
}

console.log("\nMatriz en columnas:");
for (let j = 0; j < 3; j++) {
  for (let i = 0; i < 3; i++) {
    process.stdout.write(matriz[i][j] + " ");
  }
}
console.log();
